components/collections/buttons.tsx

```diff
@@ -0,0 +1,81 @@
+"use client";
+
+import { Button } from "@/components/ui/button";
+import Link from "next/link";
+import {
+  AlertDialog,
+  AlertDialogAction,
+  AlertDialogCancel,
+  AlertDialogContent,
+  AlertDialogDescription,
+  AlertDialogFooter,
+  AlertDialogHeader,
+  AlertDialogTitle,
+  AlertDialogTrigger,
+} from "@/components/ui/alert-dialog";
+import React from "react";
+import { useDeleteCollection } from "@/collections/hooks";
+
+export const CreateCollectionButton = () => {
+  return (
+    <Button
+      className="hover:text-white rounded-sm bg-white text-black border border-gray-300"
```

Comment:
I noticed @thebeyondr was updating from grey to slate in his PR. Is that a site-wide change?

Reply:
I wasn't aware and not sure if it's a site-wide change. If it is, I don't think this should be applied to each button separately. Will bring it up in the dev checkin in an hour.